tox.ini

```diff
@@ -12,5 +12,10 @@ deps =
     requests
     pyzx
     pandas
-commands = 
+    tea-client==0.0.7
+    tea-console==0.0.5
+    typer==0.3.2
+commands =
+    pip install --upgrade https://github.com/unitaryfund/metriq-client/tarball/master
```

Comment:
Adding metriq-client dependencies and installing package from github
@WrathfulSpatula couldn't find a pypi module distribution for it, so installed the package in our env this way, and was able to create and run a`client.hello()` successfully :)
Pls let us know if there is a better installation method

Reply:
@karlaspuldaro The client needs some "TLC" before a PyPi release, but we're working on it! This looks like the right idea, for the current state of development. Glad you got it working!